core/src/services/iframe.js

```diff
@@ -197,7 +197,7 @@ class IframeClass {
     return !config.iframe.luigi.initOk;
   }
 
-  navigateIframe(config, component, node) {
+  async navigateIframe(config, component, node) {
```

Comment:
We transformed the function to async.... not whenever we call it, we transform it as a promise....So the callers (at least then do not specify to "await", the will not wait for the end of this function and it could potentially get some errors.... It is called here without await...: 
-  File: iframe.js | method setOkResponseHandler | line 159
-  File iframe| method handleRouteChange | line 390 

seeing quickly the code, I don't think we will have problems, but async function can generate some mess....

Reply:
@legteodav  I've pushed a fix which applied the `await` term on `navigateIframe`. I somehow feel it is cleaner from a syntax point of view. Thanks for point this out.